cms/server/admin/handlers/user.py

```
@@ -34,11 +34,12 @@
 from datetime import date
 
 from sqlalchemy import and_, exists
-from cms.db import Contest, Participation, Submission, Team, User
+from cms.db import Contest, Participation, Submission, Team, User, TrainingDay, TrainingProgram
 from cms.server.picture_utils import (
     process_picture_upload, PictureValidationError
 )
 from cms.server.util import exclude_internal_contests, validate_date_of_birth
+from cms.server.util import exclude_internal_contests
```

Ruff (0.14.14)

[error] Redefinition of unused `exclude_internal_contests` from line 41: `exclude_internal_contests` redefined here

Remove definition: `exclude_internal_contests`

(F811)